Project/Source/Modules/ModuleRender.cpp

```diff
@@ -43,18 +43,21 @@
 #include <math.h>
 #include <vector>
 
-static std::vector<float> ssaoGaussKernel;
+struct Kernel {
+	std::vector<float> ssaoGaussKernel;
+	std::vector<float> smallGaussKernel;
+	std::vector<float> mediumGaussKernel;
+	std::vector<float> largeGaussKernel;
+};
 
-static std::vector<float> smallGaussKernel;
-static std::vector<float> mediumGaussKernel;
-static std::vector<float> largeGaussKernel;
+static Kernel* kernelStruct = new Kernel();
```

Comment:
You could move the vectors to the module private members to avoid new/RELEASE and the auxiliary struct.

Reply:
Was it any reason to put them to static then? It could be done this way from the beginning

Reply:
Yes, there was no reason to put them static actually.